src/VolleyManagement.API/Model/GoogleUserInfo.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace VolleyManagement.API.Model
+{
+    /// <summary>
+    /// Contains user info provided by google auth to UI
+    /// </summary>
+    public class GoogleUserInfo
+    {
+        public string AccessToken { get; set; }
+
+        public string ExpiresAt { get; set; }
+
+        public string ExpiresIn { get; set; }
+
+        public string IdToken { get; set; }
+
+        public string Code { get; set; }
+
+        public string ID { get; set; }
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename property 'ID' to match camel case naming rules, consider using 'Id'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS100)